react/src/components/ImageList.tsx

```diff
@@ -0,0 +1,345 @@
+import Flex from '../components/Flex';
+import { getImageFullName } from '../helper';
+import { useBackendAIImageMetaData, useUpdatableState } from '../hooks';
+import ImageInstallModal from './ImageInstallModal';
+import { ConstraintTags } from './ImageTags';
+import ManageAppsModal from './ManageAppsModal';
+import ManageImageResourceLimitModal from './ManageImageResourceLimitModal';
+import ResourceNumber from './ResourceNumber';
+import {
+  ImageListQuery,
+  ImageListQuery$data,
+} from './__generated__/ImageListQuery.graphql';
+import {
+  AppstoreOutlined,
+  SettingOutlined,
+  VerticalAlignBottomOutlined,
+} from '@ant-design/icons';
+import { Button, Table, Tag, theme } from 'antd';
+import { AnyObject } from 'antd/es/_util/type';
+import { ColumnsType } from 'antd/es/table';
+import { ColumnType } from 'antd/lib/table';
+import graphql from 'babel-plugin-relay/macro';
+import _ from 'lodash';
+import { Key, useMemo, useState, useTransition } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useLazyLoadQuery } from 'react-relay';
+
+export type EnvironmentImage = NonNullable<
+  ImageListQuery$data['images']
+>[number];
+
+export default function ImageList() {
```

Comment:
I think it would be better to use the arrow function.

Reply:
I fixed style related issues and added the style prop

Reply:
I used function declaration to write the declaration and default export at once. can you explain why arrow function is better in react component?